### PR TITLE
CVSL-2205: Add endpoint to submit an assessment for address checks.

### PR DESCRIPTION
**Pull request by justin-gardiner**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt [hunk 1]
```diff
@@ -74,5 +74,3 @@ data class Offender(
   @Override
   override fun toString() = this::class.simpleName + "(id: $id, status: $status)"
 }
-
-fun Offender.currentAssessment(): Assessment = this.assessments.first { it.status == AssessmentStatus.NOT_STARTED || it.status === AssessmentStatus.OPTED_OUT }
```
Comment (justin-gardiner) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt:78]: not used anymore

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/OffenderResourceIntTest.kt [hunk 2]
```diff
@@ -144,12 +146,12 @@ class OffenderResourceIntTest : SqsIntegrationTestBase() {
 
       assertThat(assessment).isEqualTo(
         AssessmentSummary(
-          forename = "FIRST-4",
```
Comment (justin-gardiner) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/OffenderResourceIntTest.kt:147]: Just updating the test due to a change in the prison number being used

**Review by andrewrlee — DISMISSED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**